assembly/events-broker/configurations/broker.xml

```diff
@@ -111,25 +111,10 @@
         </security-settings>
 
         <address-settings>
-            <!-- if you define auto-create on certain queues, management has to be auto-create -->
-            <address-setting match="activemq.management#">
-                <dead-letter-address>DLQ</dead-letter-address>
-                <expiry-address>ExpiryQueue</expiry-address>
-                <redelivery-delay>0</redelivery-delay>
-                <!-- with -1 only the global-max-size is in use for limiting -->
-                <max-size-bytes>-1</max-size-bytes>
-                <message-counter-history-day-limit>10</message-counter-history-day-limit>
-                <address-full-policy>PAGE</address-full-policy>
-                <auto-create-queues>true</auto-create-queues>
-                <auto-create-addresses>true</auto-create-addresses>
-                <auto-create-jms-queues>true</auto-create-jms-queues>
-                <auto-create-jms-topics>true</auto-create-jms-topics>
-                <default-address-routing-type>ANYCAST</default-address-routing-type>
-            </address-setting>
             <!--default for catch all-->
             <address-setting match="#">
-                <dead-letter-address>DLQ</dead-letter-address>
-                <expiry-address>ExpiryQueue</expiry-address>
+                <dead-letter-address>dlq</dead-letter-address>
+                <expiry-address>expired</expiry-address>
                 <redelivery-delay>0</redelivery-delay>
```

Comment:
Missing $SYS/ prefix, is that right?

Reply:
not forgotten, it's the service broker used for authentication so for now I preferred to leave the queues without prefixes.
Do you suggest to use the prefix here (service broker) also? (in the event broker I prefer to leave namespace as it is now)

Reply:
To me it doesn't seem like a bad idea to use this separate namespace for internal messages, though I realize it's for a separate broker and adds a little overhead.  Otherwise I think the PR is fine and will go ahead and approve.

Reply:
To be honest I'm not sure about which is the best solution. I mean, the telemetry broker handles both external load (messages from clients and Camel consumers) and internal (like the dlq for example)
On the other side the event and service brokers handle only internal load now (events are meant to be used only by internal components now and the authentication service is performed by a service consumer built for this scope).
Anyway having services and events deployed in the same broker could lead to a namespace conflicts.

Reply:
I think what you are suggesting worth to be implemented now.
I'm proposing a refactoring like this one (I'm going to implement it in a new commit)
Telemetry broker:
$SYS/dlq/default
$SYS/dlq/expired

Event-broker
$SYS/svc/evt/…..

Service-broker
$SYS/svc/auth/req
$SYS/svc/auth/resp/brokerId

I think it's a good proposal but I'm looking forward for your feedback!

Reply:
I agree that this separate namespace should help avoid conflicts and provide more flexibility with how components are deployed.  I think it would also be helpful to consider now how it should be structured/used.  For the telemetry dlq for example, should it also specify the service/consumer - i.e. "$SYS/telemetry/dlq/default", etc.?

Reply:
I preferred to use MSG so the new addresses (last commit) are:

Telemetry broker
$SYS/MSG/dlq/default
$SYS/MSG/dlq/expired

Event-broker
$SYS/EVT/EVENT_ADDRESS

Service-broker
$SYS/SVC/auth/req
$SYS/SVC/auth/resp/brokerId

$SYS/SVC/dlq/default
$SYS/SVC/dlq/expired

I moved event prefix out of svc.
What do you think?

Reply:
Works for me!